code-for-canada/project-thundercat
# Submit and quit test flow - with content

# Description

Update the submit and quit test pages to have content.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

<img width="1368" alt="Screen Shot 2019-06-26 at 10 44 49 PM" src="https://user-images.githubusercontent.com/4640747/60229500-1d76fe80-9864-11e9-9ab0-85acdc466cba.png">
<img width="1371" alt="Screen Shot 2019-06-26 at 10 48 21 PM" src="https://user-images.githubusercontent.com/4640747/60230628-8f4f4800-9864-11e9-8958-f52cbe5e465c.png">


# Testing

Manual steps to reproduce this functionality:

1.  Go to the sample test.
2.  Quit the test.
3.  Submit the test.
4. Notice different confirmation pages.

# Checklist
